### PR TITLE
Feature/ComputeExecutionCondition

### LINKED ISSUE #581: create ServiceExecutionCondition
details follow

Related to #579

### PR DESCRIPTION
**Pull request by 0x3bfc**
closes #581 

Related issues: #579

### CODE REVIEW

File: contracts/conditions/ServiceExecutorCondition.sol [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        return state;
+    }
+    
+    /**
+    * @notice isTriggeredService checks whether the service is triggered or not.
+    * @param _did Decentralized Identifier (unique service/asset resolver) describes the service
+    * @param _serviceConsumer is the service consumer's address
+    * @return true if the service is triggered 
+    */
+    function isTriggeredService(
```
Comment (H34D) [inline contracts/conditions/ServiceExecutorCondition.sol]: that sounds weird.

When is this called? When the Service was triggered?

Proposals:

`isServiceTriggered`
`wasServiceTriggered`

Comment (0x3bfc) [reply contracts/conditions/ServiceExecutorCondition.sol]: >> When is this called? When the Service was triggered?

- By calling the `fulfil` method, the service execution will be triggered. However this doesn't imply that the service execution is completed or even eventually will be completed. It implies that the service provider approved that he received the request and started the execution.

- Yep `wasServiceTriggered` is much better.

**Comment by 0x3bfc** [comment]
>> serviceExecutionCondition is ambiguous as it could refer to a previous SEA that's wrapped as a condition (if a previous SEA is fulfilled, then this condition is fulfilled`)

I have followed the documentation in [OEP-12](https://github.com/oceanprotocol/OEPs/tree/master/12)

@aaitor could you please check out @diminator comment here https://github.com/oceanprotocol/keeper-contracts/pull/609#pullrequestreview-281346349

**Comment by aaitor** [comment]
> @aaitor could you please check out @diminator comment here #609 (review)

Don't worry about the naming included in OEP12. Whatever you both decide because is more clear, can be amended later in the OEP.

**Comment by 0x3bfc** [comment]
@diminator @aaitor  done

**Pull request merged**